docs/how-to/troubleshoot_deployment_issues.md

```diff
@@ -0,0 +1,199 @@
+# Troubleshoot the deployment issues
+
+This guide provides step-by-step troubleshooting actions to remediate deployment issues. We hope you don't need this guide. If you encounter an issue and aren't able to address it via this guide, please raise an issue [here][Bug Report].
+
+## 1. Terraform failed to deploy the Charmed Aether SD-Core because of configuration mismatch
+
+### Symptoms
+
+Terraform may fail to deploy the Charmed Aether SD-Core modules. When the `terraform apply -auto-approve` command is run, the deployment fails with the wrong integration endpoints or the wrong object attributes etc. as following:
+
+```console
+$ terraform apply -auto-approve 
+╷
+│ Error: Unsupported attribute
+│ 
+│   on main.tf line 54, in resource "juju_integration" "cu-nms":
+│   54:     endpoint = module.sdcore.fiveg_identity_endpoint
+│     ├────────────────
+│     │ module.sdcore is a object
+│ 
+│ This object does not have an attribute named "fiveg_identity_endpoint".
+```
+
+### Recommended Actions
+
+[Validate the Terraform modules][Validate Terraform Configuration]:
+
+```shell
+$ terraform validate
+Success! The configuration is valid.
+```
+
+The command should not fail. If it does, please open a [bug report][Bug Report].
+
+## 2. Charmed Aether SD-Core charms stuck at the Waiting/Error status
```

Comment:
Similar comment as above - if the recommended action is opening a bug report, should that go under `Troubleshooting`?

Reply:
I do not think it is a bad idea to tell users to create a bug report.  This document  picks up some cases that users may generally encounter and we recommend some actions to do. Some issues are environmental and the users may solve it by following the guidance but the others requires developer effort.

Reply:
Yeah I agree with @Gmerold on this one, there is no troubleshooting action here.

This doc should contain steps of things to do for users to fix their own problems.

Reply:
The cases that does not provide solution is removed from the doc.